### PR TITLE
Fix #121: Removes DOCKER_MACHINE_NAME from `env docker` command output

### LINKED ISSUE #121: Remove DOCKER_MACHINE_NAME from service-manager env docker call
There seems to be no reason to have this variable in the generated output. At least not for the Host to VM docker communication. See also https://github.com/projectatomic/vagrant-service-manager/issues/21#issuecomment-199194003

### PR DESCRIPTION
**Pull request by navidshaikh**
Fixes #121

### CODE REVIEW

**Comment by hferentschik** [comment]
+1 from my side. As said technically, this variable is not needed from a Docker communication point of view. In fact when it comes to docker-machine it was also added quite late. I found this discussion - https://github.com/docker/machine/issues/925. Exporting the name, helps to distinguish different Docker connection in separate consoles, eg via using the name in the prompt. The difference here is, that in the case of docker-machine we are actually dealing with human assigned names, which can indeed help. In our case we are just writing out the Vagrant machine id which is not so helpful.

In the long run it makes sense to be able to control the CDK(s) from another directory than the one the _Vagrantfile_ is located. At that stage one would need a name/id. But right now we don't have even this functionality and I think we need to think through first on how such a feature would look like. For example, could we also assign/use human readable name.

Bottom line, as long as we only have one CDK and we can only control it from within the directory the _Vagrantfile_ resides, we are imo better off not having this variable at all and introduce it when we have a need for it.

**Comment by navidshaikh** [comment]
> In our case we are just writing out the Vagrant machine id which is not so helpful.

Once we have #98 implemented, having the machine id in the output will help since the machine id displayed is same as one sees in `vagrant global-status` output. 

> Bottom line, as long as we only have one CDK and we can only control it from within the directory the Vagrantfile resides, we are imo better off not having this variable at all and introduce it when we have a need for it.

+1

@hferentschik 
Also I would like to confirm other tooling parsing the output of `env docker` should not break if we have changed bits in the output.

**Comment by navidshaikh** [comment]
@robstryker ping could you  also check? here is the  updated output

```
$vagrant service-manager env docker
# Copying TLS certificates to /home/nshaikh/work/src/vagrant-service-manager/.vagrant/machines/default/virtualbox/docker
# Set the following environment variables to enable access to the
# docker daemon running inside of the vagrant virtual machine:
export DOCKER_HOST=tcp://172.28.128.4:2376
export DOCKER_CERT_PATH=/home/nshaikh/work/src/vagrant-service-manager/.vagrant/machines/default/virtualbox/docker
export DOCKER_TLS_VERIFY=1
export DOCKER_API_VERSION=1.20
# run following command to configure your shell:
# eval "$(vagrant service-manager env docker)"

```

**Comment by hferentschik** [comment]
> Once we have #98 implemented, having the machine id in the output will help since the machine id displayed is same as one sees in vagrant global-status output. 

Sure, but what I am saying here is that if possible we should be able to use a string id provided by the user instead of a bunch of numbers. That's what docker-machine does as well. Having a VM name of _local_cdk_ is helpful and easy. Having to use 953134, not so much. We need to investigate how we can use a user specified name.

**Comment by navidshaikh** [comment]
> but what I am saying here is that if possible we should be able to use a string id provided by the user instead of a bunch of numbers. That's what docker-machine does as well. 

Understood!

> Having a VM name of local_cdk is helpful and easy. Having to use 953134, not so much. We need to investigate how we can use a user specified name.

+1 here is the tracker for same https://github.com/projectatomic/vagrant-service-manager/issues/124

**Comment by robstryker** [comment]
JBT has not made use of DOCKER_MACHINE_NAME in any form, to the best of my knowledge.

**Comment by brgnepal** [comment]
LGTM

**Comment by navidshaikh** [comment]
@bexelbie : This is ready to be merged.

**Comment by praveenkumar** [comment]
LGTM
